ignite/metrics/regression/pearson_correlation.py

```diff
@@ -0,0 +1,123 @@
+from typing import Callable, Tuple, Union
+
+import torch
+
+from ignite.exceptions import NotComputableError
+from ignite.metrics.metric import reinit__is_reduced, sync_all_reduce
+
+from ignite.metrics.regression._base import _BaseRegression
+
+
+class PearsonCorrelation(_BaseRegression):
+    r"""Calculates the `Pearson correlation coefficient
+        <https://en.wikipedia.org/wiki/Pearson_correlation_coefficient>`_.
```

Comment:
Something is wrong with this line: 
```
 /home/runner/work/ignite/ignite/ignite/metrics/regression/pearson_correlation.py:docstring of ignite.metrics.regression.pearson_correlation.PearsonCorrelation:1: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/home/runner/work/ignite/ignite/docs/source/metrics.rst:378:<autosummary>:1: WARNING: Inline interpreted text or phrase reference start-string without end-string.
```
https://github.com/pytorch/ignite/actions/runs/8481423782/job/23238731127?pr=3212

Let's try this: 
```suggestion
    r"""Calculates the 
    `Pearson correlation coefficient <https://en.wikipedia.org/wiki/Pearson_correlation_coefficient>`_.
```